pkg/operators/cli/clioperator.go

```diff
@@ -237,7 +276,11 @@ func (o *cliOperatorInstance) PreStart(gadgetCtx operators.GadgetContext) error
 			formatter := p.GetTextColumnsFormatter()
 
 			if hasFields {
-				err := formatter.SetShowColumns(strings.Split(fields, ","))
+				parseFields, err := parseFields(fields, defCols)
```

Comment:
nit: 
```suggestion
				parsedFields, err := parseFields(fields, defCols)
```

(to avoid having a function and a variable with the same name)